src/styles/_catalog.scss

```diff
@@ -9,14 +11,31 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-size;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
 }
+
+@media (max-width: 1023px) {
```

Comment:
Just a suggestion - you can move media to variables and the use `@each in` loop to create responsive)